sdk/src/main/java/io/radar/sdk/Radar.kt

```diff
@@ -2550,66 +2560,16 @@ object Radar {
             return
         }
 
-        trackOnce(location, object : RadarTrackCallback {
-            override fun onComplete(status: RadarStatus, location: Location?, events: Array<RadarEvent>?, user: RadarUser?) {
-                if (status != RadarStatus.SUCCESS || location == null) {
-                    handler.post {
-                        callback.onComplete(status)
-                    }
-
-                    return
-                }
-
-                logConversion(name, data, location, events, user, callback)
-            }
-        })
-    }
-
-    /**
-     * Logs a conversion with a manually provided location.
-     *
-     * @see [](https://radar.com/documentation/api#send-a-custom-event)
-     *
-     * @param[name] The name of the conversion.
-     * @param[location] The location of the conversion.
-     * @param[metadata] The metadata associated with the conversion.
-     * @param[block] A block callback.
-     */
-    @JvmStatic
-    fun logConversion(
-        name: String,
-        location: Location,
-        data: JSONObject?,
-        block: (status: RadarStatus, location: Location?, events: Array<RadarEvent>?, user: RadarUser?) -> Unit
-    ) {
-        logConversion(name, location, data, object : RadarLogConversionCallback {
-            override fun onComplete(
-                status: RadarStatus,
-                location: Location?,
-                events: Array<RadarEvent>?,
-                user: RadarUser?
-            ) {
-                block(status, location, events, user)
-            }
-        })
-    }
-
-    @JvmStatic
-    private fun logConversion(
-        name: String,
-        data: JSONObject?,
-        location: Location?,
-        events: Array<RadarEvent>?,
-        user: RadarUser?,
-        callback: RadarLogConversionCallback
-    ) {
         apiClient.sendEvent(
             name,
-            data,
-            user,
-            events,
+            metadata,
             object : RadarApiClient.RadarSendEventApiCallback {
-                override fun onComplete(status: RadarStatus, res: JSONObject?, events: Array<RadarEvent>?) {
+                override fun onComplete(
+                    status: RadarStatus,
+                    res: JSONObject?,
+                    event: RadarEvent?,
+                    user: RadarUser?
```

Comment:
`RadarUser` is not passed back from the response as a top level key. It is nested in the event so we should remove it
![image](https://user-images.githubusercontent.com/61531269/222760154-d48e572c-d06e-47ac-8a3d-e96ddce7c548.png)